### PR TITLE
Remove overwrite in webgl context object

### PR DESCRIPTION
**Pull request by pandamicro**
Re: cocos-creator/fireball#4598

Changes proposed in this pull request:
 * Remove overwrite in webgl context object to fix rendering artifacts in some devices like Vivo or Smartisan

> Makes sure these boxes are checked before submitting your PR - thank you!
>
- [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
- [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
- To official teams:
  - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
  - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)

@cocos-creator/engine-admins

### CODE REVIEW

File: cocos2d/core/CCDrawingPrimitivesWebGL.js [hunk 1]
```diff
@@ -85,12 +85,12 @@ cc.DrawingPrimitiveWebGL = cc._Class.extend(/** @lends cc.DrawingPrimitiveWebGL#
         var glContext = this._renderContext;
         this._shader.use();
         this._shader.setUniformForModelViewAndProjectionMatrixWithMat4();
-        glContext.enableVertexAttribArray(macro.VERTEX_ATTRIB_FLAG_POSITION);
+        cc.gl.enableVertexAttribArray(macro.VERTEX_ATTRIB_FLAG_POSITION);
```
Comment (jareguo) [inline cocos2d/core/CCDrawingPrimitivesWebGL.js:88]: 为什么有的地方改成了 cc.gl，有的地方改成了 ccgl... 区别是什么

**Review by jareguo — CHANGES_REQUESTED** [review]

File: cocos2d/core/CCDrawingPrimitivesWebGL.js (hunk 1 above)
Comment (pandamicro) [reply cocos2d/core/CCDrawingPrimitivesWebGL.js:88]: 只是临时变量方便存取，所有 ccgl 的使用前面都有 ccgl = cc.gl

cc 本身内部属性过多，所以预存一下会比较快

Comment (pandamicro) [reply cocos2d/core/CCDrawingPrimitivesWebGL.js:88]: 另外，CCDrawingPrimitivesWebGL.js 这里面没有优化是因为，目前没有用到这个东西，所以就不管了

**Review by jareguo — APPROVED** [review]

File: cocos2d/core/CCDrawingPrimitivesWebGL.js (hunk 1 above)
Comment (jareguo) [reply cocos2d/core/CCDrawingPrimitivesWebGL.js:88]: OK~

**Pull request merged**